src/index.html

```diff
@@ -9,5 +9,16 @@ <h1>Hello, World!</h1>
             <li><a href="https://github.com/kalispell-software-crafters/kalispell-software-crafters.github.io">If you're looking for something to help with we have a list of items to do within our Readme to flesh out this page. It's great for beginners who want to particiapte in things like Hacktoberfest!</a></li>
         </ul>
         <p>We look forward to seeing you at a future meeting!</p>
+        <h2>Beginners</h2>
+        <p>For beginners, try checking out freeCodeCamp.org, code.org, or Khan
+            Academy to get started in learning more about computer programming.
+            There's even a local group on Facebook to get help from others
+            (search for freeCodeCamp Kalispell -
+            https://www.facebook.com/groups/free.code.camp.kalispell.mt/ ).
+            You should also ask us about our Slack group! You're more than
```

Comment:
One other thing I'll need to update on the meetup page too, but could you change `Slack` to `Slack/Discord`?